### PR TITLE
Remove FieldSet Divider

### PR DESCRIPTION
**Pull request by rkuykendall**
## Screenshots
**Mighty:**
image

**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by SHA-4** [comment]
Could you rename this PR to something more appropriate? `FieldSet` is already being used in `Form`.

Also, does removing the divider change anything visually? I don't see a difference in the before and after pictures

File: src/building-blocks/FieldSet.tsx [hunk 1]
```diff
@@ -25,11 +25,13 @@ export default class FieldSet extends Component<IProps> {
       ;
 
     return (
-      <Antd.Row {...rowProps} className={cx(className)}>
-        <Legend fieldSet={fieldSet} />
+      <div className={cx('fields-ant-field-set', className)}>
```
Comment (SHA-4) [inline src/building-blocks/FieldSet.tsx]: I'm not sure what the purpose of a wrapper div would be here. Any reason in particular?

Comment (rkuykendall) [reply src/building-blocks/FieldSet.tsx]: Just styling, so you can add horizontal borders without messing with the styling / margins on Rows. This is what we use now, and from what I can tell it wouldn't be wise to do the bordering on the Row component.

Comment (rkuykendall) [reply src/building-blocks/FieldSet.tsx]: Could be removed if we find out otherwise after style changes. Doesn't make a difference though.

Comment (SHA-4) [reply src/building-blocks/FieldSet.tsx]: Sounds good

Comment (SHA-4) [reply src/building-blocks/FieldSet.tsx]: I would recommend having a const `CLASS_PREFIX = 'fields-ant'` or something similar for when we add classes to other components inevitability

**Pull request merged**